.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "codelabz-dev"
+    "default": process.env.REACT_APP_FIREBASE_PROJECT_ID
```

Comment:
I have a doubt here. Is `.firebaserc` file read by React? I don't think so. It is created by 

```bash
$ firebase init
```
command and it is read only by `firebase` cli. So changing this to an env variable is ill-advised I think. But I need a second opinion on this. @thisuraseniya, @shehand?

Reply:
let me see if get any documentation or not. maybe others can help in this case.

Reply:
I'm also having the same doubt. I think this will affect to GitHub actions(if we have them here to deploy) also since firebaserc is used in GitHub actions in firebase deployment. Better to revert this change back to original.

Reply:
ok changing it back.